site/docs/guides/contributor/release-process/index.md

```diff
@@ -0,0 +1,33 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project 2020. -->
+
+New releases can be created by Egeria maintainers that have the appropriate access on each GitHub repository.
+
+Releases are published to [Maven Central :material-dock-window:](https://oss.sonatype.org){ target=maven }.
+
```

Comment:
Container images are also published to https://quay.io/odpi and https://docker.io/odpi